src/collections/blog/2022/2022-20-06-managing-multiple-clusters-with-meshery/index.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title:  Multi-Cluster Kubernetes Management with Meshery
+subtitle: Wrangling your services one cluster at-a-time
+date: 2022-07-02 08:00:00 -0630
+author: Ashish Tiwari
+description: "Manage all of your Kubernetes clusters with the cloud native management plane, Meshery. Learn how Meshery makes connecting, discovering, and configuring multiple clusters a breeze."
+thumbnail: ./multi-cluster-kubernetes-management-with-meshery.png
+category: "Meshery"
+tags:
+ - Meshery
+ - Multi-cluster
+ - Kubernetes
+type: Blog
+published: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import Switcher from "./context-switcher.png";
+import Deploy from "./deploy-modal.png";
+import Cluster2 from "./meshmap-cluster2.png";
+import Cluster1 from "./meshmap-cluster1.png";
+import Settings from "./settings.png";
+import Cluster from "./cluster-mgmt.png";
+import Flush from "./flush-meshsync.png";
+import CoreArch from "./meshery-core-architecture.png";
+import { Link } from "gatsby";
+
+<BlogWrapper>
+
+<div class="intro">
+<p>
+From multi-mesh to now multi-cluster, <Link to="/meshery">Meshery</Link> is continuously expanding its capability to give developers, operators, and security engineers more control over their infrastructure. In this post, we'll take a look behind the scenes at how each component in Meshery's architecture plays a role in the management of many Kubernetes clusters.
+</p>
+</div>
+
+## Philosophy behind Meshery's multi-cluster management approach
```

Comment:
There's not much philosophy offered as a prelude, but more a straight jump to behind-the-scenes concepts that readers have no context for. Please lead the reader in first.